fiftyone/utils/torch.py

```
@@ -1460,6 +1468,205 @@ def _is_string_array(targets):
+                        samples.values(field_name), local_process_group
+                    )
+                else:
+                    # don't need to pass actual data if not local rank 0
+                    # we read it from shared memory anyways
+                    self.cached_fields[field_name] = TorchShmSerializedList(
+                        [], local_process_group
+                    )
+
+    @property
+    def samples(self):
+        if self._samples == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)